azkaban-execserver/src/main/java/azkaban/execapp/ServerStatisticsServlet.java

```diff
@@ -73,7 +73,7 @@ protected void fillRemainingMemoryPercent(ExecutorInfo stats){
         && new File("/bin/grep").exists()
         &&  new File("/proc/meminfo").exists()) {
     java.lang.ProcessBuilder processBuilder =
-        new java.lang.ProcessBuilder("/bin/bash", "-c", "/bin/cat /proc/meminfo | grep -E \"MemTotal|MemFree\"");
+        new java.lang.ProcessBuilder("/bin/bash", "-c", "/bin/cat /proc/meminfo | grep -E \"MemTotal|MemFree|Buffers|Cached|SwapCached\"");
```

Comment:
nit: 
I think it will be better to have strict grep to avoid unintentional string match as currently Cached will also get you SwapCached. We can use start of line and ":" to strict
e.g. grep -E \"^MemTotal:|^MemFree:|^Buffers:|^Cached:|^SwapCached:\